src/hooks/useLeagueData.ts

```
@@ -109,37 +202,123 @@ export const useLeaguePrizes = (leagueType: string) => {
+        // Load prizes from JSON file based on league type
+        const loadPrizesFromFile = async () => {
+          try {
+            let prizesData: any[] = [];
+
+            switch (leagueType) {
+              case "premier-league":
+              case "premier":
+                // Import the Premier League prizes JSON file
+                const premierPrizes = await import(
+                  "@/data/premier-league/prizes.json"
+                );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)